src/feeds/feed-fr.njk

```diff
@@ -22,7 +22,7 @@
   <link href="{{ permalink | absoluteUrl(metadata.feedUrl) }}" rel="self"/>
   <link href="{{ metadata.url }}"/>
   <updated>{{ collections.articles_fr | getNewestCollectionItemDate | dateToRfc3339 }}</updated>
```

Comment:
```suggestion
  <updated>{{ collections.articles_fr | getNewestCollectionItemDate | jsDate | dateToRfc3339 }}</updated>
```

Reply:
Same here @Guillaum31

Reply:
Close because this bug was splitted in another issue, see #708 for more informations.